docs.md

```
@@ -41,6 +41,8 @@ if (result.status) {
 }
 ```
 
+> If you get confused when reading this documentations, **just read the variables or functions section's **
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)